### ISSUE TITLE
switcher touch is detected but commands not working

### ISSUE THREAD

**Issue by daniilpevni** [TomerFi/aioswitcher#725]
### What happened?

switcher touch (hot water controller - boiler) is discovered with discover_devices.py -t 1 but all attempt to run commands against it fail (attempted, on / off / get_status).

### Module Version

dev branch

### Device Type

Switcher Touch (V3)

### Firmware Version

5.15

### Relevant log output

```shell
running python discover_devices.py -t 1  

{   'auto_shutdown': '01:30:00',
    'device_id': 'xxxxxx',
    'device_state': <DeviceState.OFF: ('00', 'off')>,
    'device_type': <DeviceType.TOUCH: ('Switcher Touch', '030b', 1, <DeviceCategory.WATER_HEATER: 1>)>,
    'electric_current': 0.0,
    'ip_address': 'xxx.xxx.xx.xx',
    'last_data_update': datetime.datetime(2024, 1, 5, 9, 37, 53, 651814),
    'mac_address': 'xx:xx:xx:xx:xx:xx',
    'name': 'xxxx',
    'power_consumption': 0,
    'remaining_time': '00:00:00'
}

but all attempts to run commands against it fail:
e.g:
python control_device.py get_state -d xxxx -i xxx.xxx.xx.xx

Traceback (most recent call last):
  File "/Users/xxxx/PycharmProjects/aioswitcher/src/aioswitcher/api/__init__.py", line 369, in get_state
    response = SwitcherStateResponse(state_resp)
  File "<string>", line 4, in __init__
  File "/Users/xxxx/PycharmProjects/aioswitcher/src/aioswitcher/api/messages.py", line 200, in __post_init__
    self.state = parser.get_state()
  File "/Users/xxxxx/PycharmProjects/aioswitcher/src/aioswitcher/api/messages.py", line 85, in get_state
    return states[hex_state]
KeyError: ''

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/xxxx/PycharmProjects/aioswitcher/scripts/control_device.py", line 429, in <module>
    asyncio.run(get_state(args.device_id, args.ip_address, args.verbose))
  File "/Users/xxxx/anaconda3/envs/electra/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Users/xxxx/anaconda3/envs/electra/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/xxxx/PycharmProjects/aioswitcher/scripts/control_device.py", line 298, in get_state
    printer.pprint(asdict(await api.get_state(), verbose))
  File "/Users/xxxx/PycharmProjects/aioswitcher/src/aioswitcher/api/__init__.py", line 373, in get_state
    raise RuntimeError("get state request was not successful") from ve
RuntimeError: get state request was not successful

after doing some debugging what i see is that after self._login() function pass (it does pass) there is attempt to send get_state package but the response is b'' (empty)

(worth mentioning that the hot water controll, does work with switchers app well so there is no problem with the device)
```

**Comment by daniilpevni** [comment]
Here is a solution:

1. upgrade firmware to 5.15 (or later) 
3. delete the device from the switcher app (and any shared)
4. change the switcher app to work in LOCAL mode (on main screen go to settings - there is a toggle "Local")
5. do clean device onboarding again on switcher app
6. The API should work now.

(apparently when device is onboarded when the app is not in "Local" it adds something to the headers that breaks the headers implemented in this package)

I am leaving the bug open in case the maintainer want to add the "non local" onboarding options support, or add it to the documentation for Switcher Touch (V3)

**Comment by YogevBokobza** [comment]
Fixed in https://github.com/TomerFi/aioswitcher/pull/724

**Comment by TomerFi** [comment]
> Fixed in #724

@YogevBokobza if you add "fixes: #725" to the PR body of #724. This issue should be closed automatically after a merge.

**Comment by TomerFi** [comment]
Fixed in [3.4.1](https://github.com/TomerFi/aioswitcher/releases/tag/3.4.1).
